apps/backend/src/applications/applications.service.ts

```diff
@@ -97,6 +103,24 @@ export class ApplicationsService {
     return apps;
   }
 
+  async findAllCurrentApplications(): Promise<GetAllApplicationResponseDTO[]> {
+    const applications = await this.applicationsRepository.find({
+      where: {
+        year: getCurrentYear(),
+        semester: getCurrentSemester(),
+      },
+      relations: ['user'],
+    });
+
+    const dtos: GetAllApplicationResponseDTO[] = [];
```

Comment:
⛏️ Can we do something like:
```tsx
const dtos: GetAllApplicationResponseDTO[] = applications.map((app) => app.toGetAllApplicationResponseDTO());
```
?

Reply:
Yep, this is a more succinct way to write the code - fix pushed!